keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -276,16 +333,45 @@ internal class ClusterHandlerTests : JUnit5Minutests {
           .containsKey("us-west-2")
       }
 
-      test("annealing a diff creates a new server group") {
+      test("annealing a new cluster only uses a createServerGroup stage if it has no scaling policies") {
+        runBlocking {
+          upsert(
+            resource,
+            ResourceDiff(
+              serverGroups.map {
+                it.copy(scaling = Scaling(), capacity = Capacity(2, 2, 2))
+              }.byRegion(),
+              emptyMap()))
+        }
+
+        val slot = slot<OrchestrationRequest>()
+        coVerify { orcaService.orchestrate("keel@spinnaker", capture(slot)) }
+
+        expectThat(slot.captured.job.size).isEqualTo(1)
+        expectThat(slot.captured.job.first()) {
+          get("type").isEqualTo("createServerGroup")
```

Comment:
Given the description, I was expecting this test to check that, if you _did_ have a scaling policy, you wouldn't find this job in here...

Reply:
I'm not sure how to process this comment but I think you were confusing `createServerGroup` with other orca stage types?

Reply:
Sorry, let me clarify. I meant, since the test description is "only uses a createServerGroup stage if it has no scaling policies", the more logical check for me would be to verify that, in the presence of scaling policies, you would _not_ find a `createServerGroup` job in the orchestration call. But I think I'm getting a bad rap for always modeling things in the negative form, so take this with a grain of salt. 😄